src/pykeen/models/unimodal/rotate.py

```diff
@@ -117,7 +118,7 @@ def interaction_function(
         )
         # Workaround until https://github.com/pytorch/pytorch/issues/30704 is fixed
```

Comment:
This issue claims it's fixed (though I've noticed PyTorch has a bad habit of merging PRs then immediately reverting them, so issues have very little integrity)

Reply:
@mberr ping

Reply:
If this found its way into the release, we can replace

```python
scores = -linalg.vector_norm(diff.view(diff.shape[:-2] + (-1,)), dim=-1)
```
by

```python
scores = -linalg.vector_norm(diff, dim=(-2, -1))
```

Reply:
> (though I've noticed PyTorch has a bad habit of merging PRs then immediately reverting them, so issues have very little integrity)

merged means merged to `master`. torch splits a release branch relatively early from master, and then only transfers some parts, cf. https://github.com/pytorch/pytorch#releases-and-contributing and https://github.com/pytorch/pytorch/issues/58518

Reply:
This was only necessary in the RotatE code. I updated it in 4c2a5929